src/api/tables.ts

```diff
@@ -1,15 +1,91 @@
 import { Router } from 'express'
 
 import sql = require('../lib/sql')
-const { tables } = sql
+const { columns, grants, pk_list, relationships, tables } = sql
```

Comment:
Perhaps we should rename `pk_list` to `primary_keys` for consistency

Reply:
👍 Agreed